api/v1alpha1/clustertemplate_types.go

```diff
@@ -29,19 +29,39 @@ type ClusterTemplateSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// Name defines a Human readable name of the Template.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Name",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	Name string `json:"name"`
+	// Description defines a Human readable description of the Template.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Description",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	Description string `json:"description,omitempty"`
+	// Version defines a version or generation of the resource as defined by its provider.>
```

Comment:
It looks like some of these descriptions have a `>` at the end. Was this intended? :thinking:

Reply:
Fixed